src/metakb/query.py

```diff
@@ -708,3 +714,66 @@ def _get_therapeutic_agent(in_ta_params: dict) -> TherapeuticAgent:
 
         ta_params["extensions"] = extensions
         return TherapeuticAgent(**ta_params)
+
+    async def batch_search_studies(
+        self,
+        variations: list[str] | None = None,
+    ) -> BatchSearchStudiesService:
+        """Fetch all studies associated with any of the provided variation description
+        strings.
+
+        Because this method could be expanded to include other kinds of search terms,
+        ``variations`` is optionally nullable.
```

Comment:
Same comment as above. Can we have it default to empty list?

Reply:
We definitely can't use mutable defaults here, right?